### ISSUE TITLE
Opening and Closing Gallery Crashes Game

### ISSUE THREAD

**Issue by FikysTM** [bunfan/beat-banger-public#204]
### Discord Tag

@fikys_tm

### Issue Description

I didn't understand how it works, but when I go in and play any of the songs, the field of victory or defeat when entering the store, the game crashes. I also noticed that frequent panel switching, music --> pictures --> cutscenes, then the game may crash if you often go in and out

### OS

Windows

### Version

3.34 dev1 (Steam)

### Relevant log output

```shell
I didn't understand how it works, but when I go in and play any of the songs, the field of victory or defeat when entering the store, the game crashes. I also noticed that frequent panel switching, music --> pictures --> cutscenes, then the game may crash if you often go in and out
```

**Comment by Stefan-Mitric** [comment]
Can you @FikysTM provide a video?

**Comment by 1MPexe** [comment]
It's the gallery apparently. There is a big lag and the possibility of it crashing if you go in and out of it. I'll provide video.

**Comment by 1MPexe** [comment]
> It's the gallery apparently. There is a big lag and the possibility of it crashing if you go in and out of it. I'll provide video.


https://github.com/bunfan/beat-banger-public/assets/89138073/08fc35fb-6c9f-4d98-a50d-3a8dc8873569

**Comment by 1MPexe** [comment]
as for the other thing they mentioned I'm not too sure of.

**Comment by komdog** [comment]
@1MPexe This could possibly be linked to #203

**Comment by komdog** [comment]
Closing this issue because I think it was fixed along with issue #203
